application/src/main/java/benchmark/datagenerator/apps/GS/TPGTxnGenerator/GSTPGDynamicDataGenerator.java

```diff
@@ -49,21 +51,78 @@ public GSTPGDynamicDataGenerator(DynamicDataGeneratorConfig dynamicDataConfig) {
     }
 
     @Override
-    public void switchConfiguration() {
-        State_Access_Skewness = dynamicDataConfig.State_Access_Skewness;
-        NUM_ACCESS = dynamicDataConfig.NUM_ACCESS;
-        Ratio_of_Transaction_Aborts = dynamicDataConfig.Ratio_of_Transaction_Aborts;
-        Ratio_of_Overlapped_Keys = dynamicDataConfig.Ratio_of_Overlapped_Keys;
-        Transaction_Length = dynamicDataConfig.Transaction_Length;
-
-        int nKeyState = dynamicDataConfig.getnKeyStates();
-        int MAX_LEVEL = 256;
-        for (int i = 0; i < nKeyState; i++) {
-            idToLevel.put(i, i% MAX_LEVEL);
+    public void tranToDecisionConf() {
```

Comment:
The name is a bit confusing. In general, for naming style, I would suggest to use full names to describe the method concisely, abbreviations makes the code hard to read. By the way, it would be better to add comments for the important methods.